apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -159,6 +159,7 @@ const ADDRESS_ATTRIBUTE: string = "addresses";
 const HOME_ATTRIBUTE: string = "home";
 const BULK_ID: string = "bulkId";
 const FILE_IMPORT_TIMEOUT: number = 60000; // 1 minutes.
+const usernameRegexErrorCode: string = "31301";
```

Comment:
Shall we move this to the constants [1] folder?

[1] https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/users/constants/user-management-constants.ts

Reply:
Moved